src/core/client/stream/common/scrollToBeginning.ts

```diff
@@ -12,6 +12,8 @@ function scrollToBeginning(
     } else {
       window.scrollTo({ top: getElementWindowTopOffset(window, tab) });
     }
+    // set keyboard focus to Comments button for accessibility
+    tab.getElementsByTagName("button")[0].focus();
```

Comment:
thanks for commenting this, the reason for this would have likely been lost to time after a couple months!